Code/Editor/AzAssetBrowser/AzAssetBrowserWindow.h

```diff
@@ -79,6 +87,10 @@ protected slots:
 private Q_SLOTS:
     void SelectionChangedSlot(const QItemSelection& selected, const QItemSelection& deselected) const;
     void DoubleClickedItem(const QModelIndex& element);
+
+private:
+    //Workaround in order to preserve the old size when docking.
+    QSize m_oldSize;
```

Comment:
nit: As this is in a header, prefer an API style comment like this: `QSize m_oldSize; //!< Workaround in order to preserve the old size when docking.`